### PR TITLE
fix #669 rename concerto_configs group column to category

### LINKED ISSUE #669: ActiveRecord::StatementInvalid: PG::Error: ERROR: syntax error at or near "group" LINE 1: ...ncerto_configs" WHERE (hidden IS NULL) ORDER BY `group`, se... ^ : SELECT DISTINCT "concerto_configs".* FROM "concerto_configs" WHERE (hidden IS NULL) ORDER BY `g
Check full details on Airbrake: https://concerto.airbrake.io/groups/66071731?auth_token=3938011c0f8c9e57e2fb7b73a8c4dd5ee5b5559f

```
[PROJECT_ROOT]/vendor/bundle/ruby/2.0.0/gems/activerecord-3.2.14/lib/active_record/connection_adapters/postgresql_adapter.rb:1163:in `exec'
[PROJECT_ROOT]/vendor/bundle/ruby/2.0.0/gems/activerecord-3.2.14/lib/active_record/connection_adapters/postgresql_adapter.rb:1163:in `exec_no_cache'
[PROJECT_ROOT]/vendor/bundle/ruby/2.0.0/gems/activerecord-3.2.14/lib/active_record/connection_adapters/postgresql_adapter.rb:660:in `block in exec_query'
[PROJECT_ROOT]/vendor/bundle/ruby/2.0.0/gems/activerecord-3.2.14/lib/active_record/connection_adapters/abstract_adapter.rb:280:in `block in log'
[PROJECT_ROOT]/vendor/bundle/ruby/2.0.0/gems/activesupport-3.2.14/lib/active_support/notifications/instrumenter.rb:20:in `instrument'
[PROJECT_ROOT]/vendor/bundle/ruby/2.0.0/gems/activerecord-3.2.14/lib/active_record/connection_adapters/abstract_adapter.rb:275:in `log'
[PROJECT_ROOT]/vendor/bundle/ruby/2.0.0/gems/newrelic_rpm-3.6.5.130/lib/new_relic/agent/instrumentation/active_record.rb:46:in `block in log_with_newrelic_instrumentation'
[PROJECT_ROOT]/vendor/bundle/ruby/2.0.0/gems/newrelic_rpm-3.6.5.130/lib/new_relic/agent/method_tracer.rb:268:in `trace_execution_scoped'
[PROJECT_ROOT]/vendor/bundle/ruby/2.0.0/gems/newrelic_rpm-3.6.5.130/lib/new_relic/agent/instrumentation/active_record.rb:43:in `log_with_newrelic_instrumentation'
[PROJECT_ROOT]/vendor/bundle/ruby/2.0.0/gems/activerecord-3.2.14/lib/active_record/connection_adapters/postgresql_adapter.rb:659:in `exec_query'
```

### PR DESCRIPTION
**Pull request by mfrederickson**
I've tested this change in sqlite3, mysql, and postgresql.  There's now a `database.yml.postgresql` example file, similar to the one for mysql-- and the gem for postgresql has been added to the Gemfile also.  I've also run rake test and all is well.

If someone can merge this when they're satisfied with it, that would be great.  Thanks.  This is for #669.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -40,6 +40,7 @@ gem 'twitter-bootstrap-rails-confirm'
 # run `bundle install --without production` to ignore MySQL.
 gem "sqlite3", :group => [:development, :test]
 gem "mysql2", :group => [:production]
+gem "activerecord-postgresql-adapter"
```
Comment (bamnet) [inline Gemfile]: I don't think we should include this.  We encourage folks to use sqlite3 & mysql but should support any ActiveRecord-based adapter.

File: db/schema.rb [hunk 2]
```diff
@@ -36,7 +36,7 @@
     t.string  "value_type"
     t.string  "value_default"
     t.string  "name"
-    t.string  "group"
```
Comment (bamnet) [inline db/schema.rb:39]: This and the version change is the only diff I'd expect here.  Any of the :null => false lines are probably going to get reverted the next time someone migrates.  Feel free to manually edit this file to make the s/group/category change and bump the version number to match.

File: Gemfile (hunk 1 above)
Comment (mfrederickson) [inline Gemfile]: Should we leave it in, but commented out (so people know it's an option), or remove it altogether?

File: db/schema.rb (hunk 2 above)
Comment (mfrederickson) [inline db/schema.rb:39]: What's the smartest way to do that?  checkout master, make the changes, checkout this branch, commit the changes?

File: Gemfile (hunk 1 above)
Comment (bamnet) [inline Gemfile]: I'd remove it altogether.  If folks need it they can should know enough to figure it out (or have it automatically installed).

File: db/schema.rb (hunk 2 above)
Comment (bamnet) [inline db/schema.rb:39]: I don't know the best git way to fix it. You could always manually wget the file, make the changes, and commit it.  I usually spot these things during a diff, then I checkout and apply manually before committing.

**Pull request merged**